SCons/Platform/Platform.xml

```diff
@@ -121,50 +121,123 @@ else:
 <cvar name="HOST_OS">
     <summary>
 <para>
-        The name of the host operating system used to create the Environment.
-        If a platform is specified when creating the Environment, then
-        that Platform's logic will handle setting this value.
-        This value is immutable, and should not be changed by the user after
-        the Environment is initialized.
+        The name of the host operating system.
+        Can be set when creating a &consenv; by passing as a keyword
+        argument in the &f-link-Environment; call; should not be
+        changed otherwise.
+        If not specified in the &f-Environment; call, will be set to
+        the defaults for the particular platform, which is in turn
+        auto-detected unless the <parameter>platform</parameter>
+        argument to &f-Environment; is given.
+</para>
+<para>
+        This &consvar; is informational and does not currently
+        affect construction activities.
 </para>
     </summary>
 </cvar>
 
-<cvar name="HOST_ARCH">
+<cvar name="TARGET_OS">
     <summary>
 <para>
-        The name of the host hardware architecture used to create the Environment.
-        If a platform is specified when creating the Environment, then
-        that Platform's logic will handle setting this value.
-        This value is immutable, and should not be changed by the user after
-        the Environment is initialized.
+        The name of the operating system compiled objects
+        created by this &consenv; should target.
+        Can be set when creating a &consenv; by passing as a keyword
```

Comment:
This is not immutable and is free to be changed. 
Though likely it should be set as desired before initializing any tools (for example compilers) which may use this info.